__tests__/app.test.js

```diff
@@ -73,4 +72,65 @@ describe('GET /api/reviews/:review_id', () => {
             expect(res.body.msg).toBe('invalid endpoint syntax')
         })
     })
-})
\ No newline at end of file
+})
+
+describe('GET /api/reviews', () => {
+    test('GET - STATUS: 200 - responds with a reviews array of all review objects with their relevant properties, excluding review_body', () => {
+        return request(app)
+        .get('/api/reviews')
+        .expect(200)
+        .then((res) => {
+            expect(res.body.reviews.length).toBe(13);
+            expect(Array.isArray(res.body.reviews)).toBe(true);
+            res.body.reviews.forEach((review) => {
+                expect(typeof review.owner).toBe('string');
+                expect(typeof review.title).toBe('string');
+                expect(typeof review.review_id).toBe('number');
+                expect(typeof review.category).toBe('string');
+                expect(typeof review.review_img_url).toBe('string');
+                expect(typeof review.created_at).toBe('string');
+                expect(typeof review.votes).toBe('number');
+                expect(typeof review.designer).toBe('string');
+                expect(review).not.toHaveProperty('review_body');
+            })
+    })
+})
+        test('GET - STATUS: 200 - the response should include a comment count which is the total count of all the comments with this review_id ', () => {
+    return request(app)
+    .get('/api/reviews')
+    .expect(200)
+    .then((res) => {
+        res.body.reviews.forEach((review) => {
+        expect(typeof review.comment_count).toBe('number')
+        expect(res.body.reviews[5].comment_count).toBe(3)
+        expect(res.body.reviews[3].comment_count).toBe(0)
+    })
+})
+        });
+        test('GET - STATUS: 200 - the default result response order should be by date in descending order ', () => {
+            return request(app)
+            .get('/api/reviews')
+            .expect(200)
+            .then((res) => {
+                const reviewDates = res.body.reviews.map(review => review.created_at);
+                expect(reviewDates).toBeSorted({ descending: true})
+            })
+        })
+        test('GET - STATUS: 400 - the client should not be allowed to input anything but the default order criteria at the moment ', () => {
+            return request(app)
+            .get('/api/reviews?order_by=asc')
+            .expect(400)
+            .then((res) => {
+                expect(res.body.msg).toBe('invalid order query');
+})
+        })
+        test('GET - STATUS: 400 - the client should not be allowed to input anything but the default sort criteria at the moment ', () => {
+            return request(app)
+            .get('/api/reviews?sort_by=owner')
+            .expect(400)
+            .then((res) => {
+                expect(res.body.msg).toBe('invalid sort query');
+})
+        })
```

Comment:
These test's aren't needed. At some point if you wanted to be able to sort_by or order_by other criteria these tests are going to fail and we should be ensuring that our tests are still able to pass as we build our functionality.